.github/workflows/push.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ release/* ]
```

Comment:
We took this out of Adapt, but really it just allows us to manually tag releases right?

Reply:
I'll remove it